src/ui/components/navigation/navigationcomponent.js

```diff
@@ -236,12 +238,31 @@ export default class NavigationComponent extends Component {
       this.refitNav();
       DOM.on(DOM.query(this._container, '.yxt-Nav-more'), 'click', this.toggleMoreDropdown.bind(this));
     }
+    DOM.queryAll(this._container, '.js-yxt-navItem').forEach((selector, index) => {
+      DOM.onClickOrEnter(selector, this.reportFollowUpQueryEvent.bind(this, index));
+    });
   }
 
   onUnMount () {
     this.unbindOverflowHandlers();
   }
 
+  reportFollowUpQueryEvent (tabIndex) {
+    const searcher = tabIndex === 0 ? Searcher.UNIVERSAL : Searcher.VERTICAL;
```

Comment:
is it always guaranteed that the universal tab will be the first one? or is that customizable (maybe with the `isFirst` option for vertical pages)?

Reply:
It's not guaranteed the first one will be universal. But in navigation handlebars, we already assume the first one is a universal page since we are sending `ALL_TAB_NAVIGATION` instead of `VERTICAL_TAB_NAVIGATION`. I will look for another way to check for this